platforms/gke/base/core/custom_compute_class/templates/manifests/gpu/h100-80gb/custom-compute-gpu-h100-80gb-x8-high.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: cloud.google.com/v1
+kind: ComputeClass
+metadata:
+  name: gpu-h100-80gb-x8-high
+spec:
+  activeMigration:
+    optimizeRulePriority: true
+  priorities:
+    # Use reservations if available, fall back to on-demand otherwise
+    - gpu:
+        count: 8
+        driverVersion: latest
+        type: nvidia-h100-80gb
+      machineType: a3-highgpu-8g
```

Comment:
Did you test with a3-ultra/H200? Just  curious because they are actually $4/hour cheaper and much more powerful.

Reply:
Not yet. Leaving this open as a reminder.

Reply:
Will address this in the next PR.